cmd/dextre/cmd/robots.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var goldenFile string
+
+// RobotsCmd represents the robots command
+var RobotsCmd = &cobra.Command{
+	Use:   "robots",
+	Short: "Monitor published robots.txt",
+	Long: `Helps with robots.txt monitoring, detecting changes from a
```

Comment:
This might be a better description of this command: `Compare the document responded by an HTTP server with a golden file`. It may also be worth renaming to `dextre compare` instead of `dextre robots`.

Reply:
What is a golden file? Is that a common phrase?

Reply:
It's a test strategy were you use a file as reference of what the output should be. I've always seen the reference files referred as golden.